tests/integ/test_udtf.py

```diff
@@ -413,6 +419,7 @@ def assert_vectorized_udtf_result(source_table: Table, udtf: UserDefinedTableFun
     )
 
 
+@pytest.mark.skipif(not is_pandas_available, reason="pandas is required")
```

Comment:
Orthogonal to your change: we should have a merge gate that does a smoke test from no-pandas environment.

Reply:
Agreed, I have a ticket here:https://snowflakecomputing.atlassian.net/browse/SNOW-873417